Test/PacketType/DrdaPacketTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿/*
+This file is part of PacketDotNet
+
+PacketDotNet is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+PacketDotNet is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with PacketDotNet.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/*
+ *  Copyright 2010 Chris Morgan <chmorgan@gmail.com>
+ */
+
+using System;
+using System.Text;
+using NUnit.Framework;
+using SharpPcap;
+using SharpPcap.LibPcap;
+using PacketDotNet;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Test.PacketType
+{
+    class DrdaPacketTest
+    {
+        [Test]
+        public void TestSelectPacket()
```

Comment:
Excellent that you've added a test here. Can you compare some fields of the decoded packets against what you'd expect them to be? Right now you are just printing the data out so if the decoding was broken and printing out incorrect values the test would still pass. I typically use wireshark to retrieve values for the unit tests here as wireshark has a ton more development time put into it than these decoders have had.

Reply:
There are lots of different drda packet, so I have to finish decode drda procotol first,than I will write more unit test case